lib/html-proofer/check/links.rb

```diff
@@ -129,10 +129,10 @@ def find_fragments(html, fragment_ids)
     html.xpath(*xpaths)
   end
 
-  IGNORABE_REL = %(canonical alternate next prev previous icon manifest apple-touch-icon)
+  SRI_REL_TYPES = %(stylesheet)
 
   def check_sri(line, content)
-    return if IGNORABE_REL.include?(@link.rel)
+    return unless SRI_REL_TYPES.include?(@link.rel)
```

Comment:
I like this but I think I have to think on it a bit more before approving. For example, `script` and `link` will also need to be checked. I'm not sure if there are more.

Reply:
So, this code should only apply to `link` tags - `script` is handled by another check, in https://github.com/gjtorikian/html-proofer/blob/master/lib/html-proofer/check/scripts.rb#L33, so is unaffected. As I mentioned, as far as I can tell from the spec, the only thing that is proactively loaded in by browsers from `link` tags are `rel='stylesheet'`, which this should cover.

Reply:
You're right! My mistake.